capstone/src/main/webapp/dashboard-loader.js

```diff
@@ -2,7 +2,21 @@ document.getElementById("signout").prepend(dynamicButton);
 window.addEventListener('authorized', getRooms);
 
 function createRoom(title, description) {
-    var roomData = JSON.stringify({"title": title, "description": description});
+    var checkboxes = document.getElementsByClassName("form-check-input");
```

Comment:
You could do something like:
`document.querySelectorAll('input[name=tag-checkboxes]:checked')`
to get just the checked ones. Then you would not have to check `checked` below.

Could could even get real fancy and do something like:
```javascript
const tagsData =
    document.querySelectorAll('input[name=tag-checkboxes]:checked')
    .map((checkbox) => checkbox.value);
```

The above selector would require adding a name attribute to your checkboxes below, or you could select by class, `.form-check-input:checked`, but I think added a semantically  meaningful name would be better.